docs/intro-events.md

```diff
@@ -16,19 +16,22 @@ Hubs is a platform where you can create private rooms and meet with friends in a
 
 Events in Hubs work best for meetups that are designed to have attendees engaged and participating actively with one another, or as a shared viewing environment for streamed video content. 
 
-If you are interested in hosting a large event with multiple tracks, you may be interested in exploring [Hubs Cloud](./hubs-cloud-getting-started.html). Hubs Cloud allows you to customize your deployment of the Hubs platform, so that you manage and maintain the system, offers configuration tools for rooms and accounts, and is better suited for larger events. For events that have larger participant interest, we recommend creating several viewing rooms and streaming the primary content to a service like [Twitch](https://twitch.tv).
+<!-- If you are interested in hosting a large event with multiple tracks, you may be interested in exploring [Hubs Cloud](./hubs-cloud-getting-started.html).  -->
+Hubs Cloud allows you to customize your deployment of the Hubs platform, so that you manage and maintain the system, offers configuration tools for rooms and accounts, and is better suited for larger events. For events that have larger participant interest, we recommend creating several viewing rooms and streaming the primary content to a service like [Twitch](https://twitch.tv).
 
 ## Setting up a Room
-Your event will need a room, which can be created from the home page at [hubs.mozilla.com](https://hubs.mozilla.com). When you create a room, you are the room's "owner", which grants you full rights and permissions within the room you have created. We recommend that you log in with a Hubs account when you create rooms, since this will ensure that you are still recorded as the room owner even if you change devices. Logged-in room owners have additional configuration tools available to them that enable more nuanced control over how visitors can interact in the space.
+Your event will need a room. When you create a room, you are the room's "owner", which grants you full rights and permissions within the room you have created. We recommend that you log in with a Hubs account when you create rooms, since this will ensure that you are still recorded as the room owner even if you change devices. Logged-in room owners have additional configuration tools available to them that enable more nuanced control over how visitors can interact in the space.
```

Comment:
Nit: I think just removing the link here is enough and this potentially provides some good info.  But since we're going to rewrite the docs anyway I'm not super invested in it either way.
```suggestion
Your event will need a room, which can be created from the home page at your instance. When you create a room, you are the room's "owner", which grants you full rights and permissions within the room you have created. We recommend that you log in with a Hubs account when you create rooms, since this will ensure that you are still recorded as the room owner even if you change devices. Logged-in room owners have additional configuration tools available to them that enable more nuanced control over how visitors can interact in the space.
```

Reply:
I think this language is more confusing.

Reply:
Okay.  Fair enough.